xp.pro

```diff
@@ -409,34 +398,61 @@ isEmpty(BOOST_LIB_SUFFIX) {
 
 isEmpty(BOOST_THREAD_LIB_SUFFIX) {
     BOOST_THREAD_LIB_SUFFIX = $$BOOST_LIB_SUFFIX
+    macx:BOOST_THREAD_LIB_SUFFIX=-mt
 }
 
 isEmpty(BDB_LIB_PATH) {
-    macx:BDB_LIB_PATH = /usr/local/BerkeleyDB.6.1/lib
+    macx:BDB_LIB_PATH=/usr/local/opt/berkeley-db@4/lib
+    windows:BDB_LIB_PATH=c:/deps/db-4.8.30.NC/build_unix
 }
 
 isEmpty(OPENSSL_LIB_PATH) {
-    macx:OPENSSL_LIB_PATH = /usr/local/ssl/lib
+    macx:OPENSSL_LIB_PATH = /usr/local/opt/openssl/lib
+    windows:OPENSSL_LIB_PATH=c:/deps/openssl-1.0.2d
 }
 
 isEmpty(BDB_LIB_SUFFIX) {
-    macx:BDB_LIB_SUFFIX = -6.1
+    macx:BDB_LIB_SUFFIX = -4.8
 }
 
 isEmpty(BDB_INCLUDE_PATH) {
-    macx:BDB_INCLUDE_PATH = /usr/local/BerkeleyDB.6.1/include
+    macx:BDB_INCLUDE_PATH=/usr/local/opt/berkeley-db@4/include
+    windows:BDB_INCLUDE_PATH=c:/deps/db-4.8.30.NC/build_unix
 }
 
 isEmpty(OPENSSL_INCLUDE_PATH) {
-    macx:OPENSSL_INCLUDE_PATH = /usr/local/ssl/include
+    macx:OPENSSL_INCLUDE_PATH=/usr/local/opt/openssl/include
+    windows:OPENSSL_INCLUDE_PATH=c:/deps/openssl-1.0.2d/include
 }
 
 isEmpty(BOOST_LIB_PATH) {
-    macx:BOOST_LIB_PATH = /usr/local/lib
+    macx:BOOST_LIB_PATH=/usr/local/opt/boost/lib
+    windows:BOOST_LIB_PATH=C:/deps/boost_1_53_0/stage/lib
 }
 
 isEmpty(BOOST_INCLUDE_PATH) {
-    macx:BOOST_INCLUDE_PATH = /usr/local/include
+    macx:BOOST_INCLUDE_PATH=/usr/local/opt/boost/include
+    windows:BOOST_INCLUDE_PATH=C:/deps/boost_1_53_0
+}
+
+isEmpty(MINIUPNPC_INCLUDE_PATH) {
+    macx:MINIUPNPC_INCLUDE_PATH=/usr/local/opt/miniupnpc/include
+    windows:MINIUPNPC_INCLUDE_PATH=C:/deps/
+}
+
+isEmpty(MINIUPNPC_LIB_PATH) {
+    macx:MINIUPNPC_LIB_PATH=/usr/local/opt/miniupnpc/lib
+    windows:MINIUPNPC_LIB_PATH=C:/deps/miniupnpc
+}
+
+isEmpty(QRENCODE_INCLUDE_PATH) {
+    macx:QRENCODE_INCLUDE_PATH=/usr/local/opt/qrencode/include
+    widnwos:QRENCODE_INCLUDE_PATH=C:/deps/qrencode-3.4.4
```

Comment:
typo

Reply:
thanks!

Reply:
typo:
+    widnwos:QRENCODE_INCLUDE_PATH=C:/deps/qrencode-3.4.4
means windows - not widnwos :)

Reply:
Don't worry. I will fix typo when will support windows build